kdb/kdb_fe/urls.py

```diff
@@ -2,5 +2,6 @@
 from . import views
 
 urlpatterns = [
-    path('', views.index)
+    path('', views.index),
+    path('/Categories/', views.index),
```

Comment:
why do we need this file?

Reply:
Do you mean this one?  path('/Categories/', views.index),
I want to access directly localhost:8000/categories/list. 
I replaced the line with   path('categories/list', views.index)  but I still get a 404 message